query-engine/connector-test-kit-rs/query-engine-tests/tests/new/relation_load_strategy.rs

```diff
@@ -125,21 +125,37 @@ mod relation_load_strategy {
 
     macro_rules! relation_load_strategy_tests_pair {
```

Comment:
It's now a quadruple and not a pair 😄 Maybe `relation_load_strategy_test_suite` or something like that?

Reply:
renamed to `relation_load_strategy_tests`